group_vars/gxconfig.yml

```diff
@@ -1013,11 +1005,24 @@ base_app_main: &BASE_APP_MAIN
   # subdomain. Defaults to "/".
   #interactivetools_base_path: /
 
-  # Map for interactivetool proxy.
+  # Map for the interactivetool proxy. Mappings are stored in a SQLite
+  # database file located on this path. As an alternative, you may also
+  # store them in any other RDBMS supported by SQLAlchemy using the
+  # option ``interactivetoolsproxy_map``, which overrides this one.
   # The value of this option will be resolved with respect to
   # <data_dir>.
   interactivetools_map: "{{ galaxy_mutable_data_dir }}/interactivetools_map.sqlite"
 
+  # Use a database supported by SQLAlchemy as map for the
+  # interactivetool proxy. When this option is set, the value of
+  # ``interactivetools_map`` is ignored. The value of this option must
+  # be a `SQLAlchemy database URL
+  # <https://docs.sqlalchemy.org/en/20/core/engines.html#database-urls>`_.
+  # Mappings are written to the table "gxitproxy" within the database.
+  # This value cannot match ``database_connection`` nor
+  # ``install_database_connection``.
+  #interactivetoolsproxy_map: null
```

Comment:
ping @kysrpex

Reply:
That's ok, keep it as it is. After Galaxy has been updated I can go through the guide and merge the [migration PR that I prepared](https://github.com/usegalaxy-eu/infrastructure-playbook/pull/1251), it will take care of changing this line.